laborator/lab-09/README.md

```diff
@@ -29,7 +29,8 @@ int a1, a2, x;
    ...
```

Comment:
Este util in commit message sa spunem cate ceva despre "De ce facem commit-ul" si "Ce face commit-ul respectiv" daca nu este absolut si rapid evident din cod. 

Altfel, vor fi foarte multe commit-uri cu titlul "Update README.md" care nu spune mare lucru.

Reply:
Asa este. La acel commit recunosc ca mi-a scapat, dar schimbasem doar niste spatii si nu am acordat importanta mesajului. Dar o sa tin cont pe viitor, mersi!